src/App/Command/Release/MakeCommand.php

```diff
@@ -169,6 +172,10 @@ protected function processPackage(Package $package): void
             $git->push();
             $git->pushTags();
 
+            if ($this->confirm('Release on GitHub?')) {
```

Comment:
Maybe modify previous question to "Push commits and tags, and release on GitHub?" and remove this confirmation?

Reply:
Good idea.